vagrant/roles/common/tasks/main.yml

```diff
@@ -12,12 +12,20 @@
 
 - name: install centos and epel repos
   command: yum -y install centos-release-gluster epel-release
+  register: task_result
+  until: task_result.rc == 0
+  retries: 5
+  delay: 1
 
 - name: setup kubernetes repo
   copy: src=kubernetes.repo owner=root group=root dest=/etc/yum.repos.d
 
 - name: install base packages
-  command: yum -y install {{ install_pkgs }}
+  command: yum --enablerepo=epel -y install {{ install_pkgs }}
```

Comment:
ooops, it is left over from me experimenting when I initially was messing with the repomd.xml clashes...let me remove it